### PR TITLE
fix: avoid blocking the event loop during import

### PR DESCRIPTION
**Pull request by bdraco**
This option is new for 2024.3. Previously all integrations had to be imported in the event loop which would block the loop while it happened.

This option is safe to add for pre 2024.3 as it will do nothing.

This could be considered `pref` but not blocking the event loop is might always considered a fix

### CODE REVIEW

File: custom_components/tesla_custom/manifest.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
   "documentation": "https://github.com/alandtse/tesla/wiki",
   "iot_class": "cloud_polling",
   "issue_tracker": "https://github.com/alandtse/tesla/issues",
```
Comment (bdraco) [inline custom_components/tesla_custom/manifest.json:24]:
```suggestion
  "import_executor": true,
  "issue_tracker": "https://github.com/alandtse/tesla/issues",
```

File: custom_components/tesla_custom/manifest.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
...
+  "import_executor": true,
```
Comment (bdraco) [inline custom_components/tesla_custom/manifest.json:25]:
```suggestion
```

**Comment by alandtse** [comment]
Thanks. This looks good to me. Please note the hassfest error due to the later keys needing to be alphabetical. I'm hoping to do a release this weekend. Waiting on #853 and this.

File: custom_components/tesla_custom/manifest.json [hunk 3]
```diff
@@ -21,6 +21,7 @@
   ],
   "documentation": "https://github.com/alandtse/tesla/wiki",
   "iot_class": "cloud_polling",
```
Comment (bdraco) [inline custom_components/tesla_custom/manifest.json:23]:
```suggestion
  "import_executor": true,
  "iot_class": "cloud_polling",
```

File: custom_components/tesla_custom/manifest.json [hunk 4]
```diff
@@ -21,6 +21,7 @@
...
+  "import_executor": true,
```
Comment (bdraco) [inline custom_components/tesla_custom/manifest.json:24]:
```suggestion
```

**Pull request merged**